src/Product/Image/Assignor.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
Well we could have an `addImage()` method on the `Product` class. Then the responsibility could be on `Product\Edit::save()`. The same way as product tags, units and prices work. See https://github.com/mothership-ec/cog-mothership-commerce/blob/develop/src/Product/Edit.php#L100